docroot/sites/obermann.uiowa.edu/modules/obermann_migrate/obermann_migrate.install

```diff
@@ -40,3 +42,76 @@ function obermann_migrate_uninstall() {
   $config_factory->getEditable('migrate_plus.migration_group.default')->delete();
   $config_factory->getEditable('migrate_plus.migration_group.sitenow_migrate')->delete();
 }
+
+/**
+ * Update article summaries from the migration.
+ */
+function obermann_migrate_update_9001(&$sandbox) {
+  // Process all nodes that have a filled field_reference.
+  if (!isset($sandbox['total'])) {
+    $query = \Drupal::entityQuery('node')
+      ->condition('type', 'article', '=')
+      ->execute();
+    $sandbox['total'] = count($query);
+    $sandbox['current'] = 0;
+    $sandbox['query'] = $query;
+
+    if (empty($sandbox['total'])) {
+      $sandbox['#finished'] = 1;
+      return;
+    }
+  }
+
+  $truncate = new TruncateHTML();
+
+  // Loop through the same nodes in batches.
+  $nodes_per_batch = 25;
+  $batch = array_slice($sandbox['query'], $sandbox['current'], $sandbox['current'] + $nodes_per_batch);
+  if (empty($batch)) {
+    $sandbox['#finished'] = 1;
+    return;
+  }
+  foreach ($batch as $nid) {
+    $node = Node::load($nid);
+    $output = $node->body->value;
```

Comment:
PHPStorm is complaining about `body` being a protected property, which is weird because it works.

Reply:
It was complaining about a similar chunk of code during the physics migration too. Not sure why.

Reply:
Just added a vardoc.

Reply:
Yay! I'll have to keep that in mind for next time. And I suppose, you know, good documentation and stuff...but no more squigglies!

Reply:
Well there is a yellow squiggle but that is not as bad. 😆